CharacterCreation/03_Feats.txt

```diff
@@ -838,7 +845,25 @@ other weapons or class abilities. Does apply to Engineer mortars.
 	also set R2 equal to the new R1.
 60:	+10% Base damage. Other characters make Shock, Reflex, and Will saves associated 
 	with your explosives with Disadvantage.
-		
+
+== Melee Expertise ==
+
+	Applies to Melee Weapons that the user has Proficiency with and Unarmed Strike. 
+Does not apply to Improvised weapons. Reminder: these bonuses are only in effect while 
+you are wielding an appropriate weapon.
+
+0:	+1 Move Speed while using the Charge action with intent to engage an enemy in 
+	Melee Combat.
+12:	May use Weapons - Melee (Int) in place of Medicine on skill checks related to 
+	triaging and healing wounds inflicted by melee weapons, nonexplosive throwing 
+	weapons, or environmental damage that approximates a melee strike.
+24:	When defending against a disarm attempt, roll maneuver checks with Advantage.
+36:	+1 Accuracy when engaging a target wielding a melee weapon (unarmed counts).
+48:	+1 Guard when engaging a target wielding a melee weapon (unarmed counts).
+60:	+10% Base damage. Gain +0.5 Critical Hit range whenever you hit a target but
+	do not score a Critical Hit. This bonus resets if you score a Critical Hit, 
+	attack a different target, are stunned, or leave Melee Combat with that target.

```

Comment:
The base damage buff can almost be factored out of the power analysis (relative to other expertise feats)  since it's on all of the 60 point bonuses. 
I had forgotten how few hits we've designed characters to take, and am quite willing to test with keeping the bonus past killing the target. Do you think it would be appropriate to also change 'leave Melee Combat' to 'spend 1 round not in Melee Combat' to allow carrying the bonus between nonadjacent enemies?

Reply:
I like how that gives a sort of "rampage" feeling to it, but I'm concerned that 1, that rampage feeling is already conferred by things like Flash Dance, and 2, that critical hits aren't what make melee weapons unique. Instead, I'm asking myself what is unique about melee weapons? Well, that you have to be next to your target to hit them... What if instead, the ultimate skill would be to be in partial cover using whomever you're in melee combat with as long as you're engaged in melee combat? Perhaps even forcing an enemy attempting to fire at you to make an explicit Luck roll when shooting at you, risking that they might hit their friend instead?

Reply:
The obvious drawback of my proposed alternative is that it doesn't find a way of keeping the good work you've done, and I'm loathe to throw away good design.

Reply:
Don't we already have a mechanic with Luck rolls when shooting into a melee, or have you been houseruling that?

Reply:
@.@ Ok lemme fix that because that should be baseline. You don't have to be all that good at melee to mess up other people's firing solutions from 30m away.